l3build.dtx

```diff
@@ -388,14 +389,14 @@
 % \item |--message| (|-m|) Text for upload announcement
 % \item |--quiet| (|-q|) Suppresses output from unpacking
 % \item |--rerun| Runs tests without unpacking/set up
-% \item |--show-log-on-error| To be used in addition to \texttt{--halt-on-error} and results
+% \item |--show-log-on-error| When in addition to \texttt{-{}-halt-on-error}, results
 %   in the full \texttt{.log} file of a failed test to be shown on the console
-% \item |--show-saves| (|-S|) When tests fail, prints the \texttt{l3build save} commands needed
-%   to regenerate the tests assuming that the failures were false negatives.
+% \item |--show-saves| (|-S|) When tests fail, prints the \texttt{l3build save ...} commands needed
+%   to regenerate the tests, assuming that the failures were false negatives.
 % \item |--shuffle| Shuffles the order in which tests run
-% \item |--stdengine| (|-s|) Run tests only with the standard engine (which can vary between
+% \item |--stdengine| (|-s|) Runs tests only with the standard engine (which can vary between
```

Comment:
Deliberate choice

Reply:
This is the only one, why the other "Runs", "Instructs", "Prints"?